system/lib/libc/musl/arch/emscripten/atomic_arch.h

```diff
@@ -5,6 +5,7 @@
 #include <emscripten.h>
 #include <emscripten/threading.h>
 
+#define a_ctz_l a_ctz_l
```

Comment:
Because https://github.com/bpowers/musl/commit/1315596b510189b5159e742110b504177bdd4932#diff-4c18db28385d6925bd8462b3a13fbce1 changed `atomic.h` to expect the archs to `#define` these, and if not it will provide the implementation. `a_cas` in particular raises an `#error` if not `#define`d